app/api/rooms.py

```diff
@@ -10,3 +11,12 @@ def get_room_or_create(profile_id):
     profile_user = User.query.get_or_404(profile_id)
     room_id = Room.get_or_create_room(current_user, profile_user)
     return jsonify(room_id)
+
+
+@bp.route('/rooms/get/recipient/<int:room_id>', methods=['GET'])
+def get_recipient_username(room_id):
+    user = Room.query.get(room_id).get_recipient(current_user)
+    if user is None:
+        return bad_request('This room is not chat')
```

Comment:
Maybe 404?

Reply:
Don't think so

Reply:
Any arguments?

Reply:
It's API request - not user interface. Seems to be better send answer with error message.